core/objects.go

```diff
@@ -143,14 +143,10 @@ type AcmeIdentifier struct {
 // CertificateRequest is just a CSR together with
 // URIs pointing to authorizations that should collectively
 // authorize the certificate being requsted.
-//
-// This type is never marshaled, since we only ever receive
```

Comment:
Why was this deleted? It still seems true. The parenthetical is confusing to me, though. Not sure how it's relevant to this struct.

Perhaps it should reference rawCertificateRequest explicitly to be more clear about what it's for?